test_aodndata/moorings/test_mooringsProductsHandler.py

```diff
@@ -54,33 +54,44 @@ def test_good_manifest(self, mock_webfeatureservice):
         self.assertCountEqual(INPUT_FILE_COLLECTION.get_attribute_list('dest_path'),
                               handler.input_file_collection.get_attribute_list('dest_path')
                               )
-        self.assertEqual(len(handler.file_collection), 7)
 
-        # check new product files
         expected_new_products = {'TEMP-aggregated-timeseries',
                                  'PSAL-aggregated-timeseries',
                                  'CHLF-aggregated-timeseries',
                                  'hourly-timeseries',
                                  'hourly-timeseries-including-non-QC'
```

Comment:
Again, this commit is inflating the amount of assertions in a single test. IMO if a test require a lot of assertions, them split the assertions into smaller testcalls to improve composability.

Reply:
This handler is doing quite a lot of things, so I make a lot of assertions to check different aspects, but they all refer to the
_same test case_ (i.e. input file). Possibly it could all be refactored to run the test case just once, then make different assertions in separate methods independently, but at this point I don't think it's worth the effort.